documentation/book/assembly-hello-world.adoc

```diff
@@ -0,0 +1,46 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+
+[id='hello-world-{context}']
+
+== Hello world example
+
+You can try this Hello world example to ensure everything is working fine for you.
+This example consists from producer and consumer which are communicating via Kafka cluster.
+
+. Setup environment
```

Comment:
This section is good, but I think it unnecessarily duplicates what we already have elsewhere and would add unnecessary effort. You should just link to the appropriate section:
* `xref:downloads-str[]`